index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Bimbala Desktop</title>
+    <!-- Include the Tailwind CSS stylesheet -->
+    <link href="https://cdn.jsdelivr.net/npm/tailwindcss@2.2.15/dist/tailwind.min.css" rel="stylesheet">
+</head>
+
+<body>
+    <section class="flex h-screen flex-col items-center md:flex-row">
+        <div class="hidden h-screen w-full bg-indigo-600 md:w-1/2 lg:block xl:w-2/3">
+            <img src="https://source.unsplash.com/random" alt="" class="h-full w-full object-cover" />
+        </div>
+
+        <div
+            class="flex h-screen w-full items-center justify-center bg-white px-6 md:mx-auto md:w-1/2 md:max-w-md lg:max-w-full lg:px-16 xl:w-1/3 xl:px-12">
+            <div class="h-100 w-full">
+                <h1 class="mb-32 text-xl font-bold leading-tight md:text-2xl">Bimbala Desktop</h1>
+
+                <form id="nameForm" class="mt-6 space-y-4">
+                    <div>
+                        <label class="block text-gray-700">Enter name:</label>
+                        <input type="text" id="name" name="name"
+                            class="mt-2 w-full rounded-lg border bg-gray-200 px-4 py-3 focus:border-blue-500 focus:bg-white focus:outline-none"
+                            autofocus autocomplete required />
+                    </div>
+
+                    <button type="submit"
+                        class="mt-6 block w-full rounded-lg bg-indigo-500 px-4 py-3 font-semibold text-white hover:bg-indigo-400 focus:bg-indigo-400">Log
+                        In</button>
+                </form>
+
+                <hr class="my-6 w-full border-gray-300" />
+
+                <!-- Display the history of the last 5 entered names -->
+                <p class="mt-2">History:</p>
+                <ul id="nameHistory" class="pl-6 list-disc font-semibold text-blue-500"></ul>
+                <hr class="my-6 w-full border-gray-300" />
+                <p class="mt-8">Need a board? <a href="https://bimbala.com/"
+                        class="font-semibold text-blue-500 hover:text-blue-700">Create one</a></p>
+                <img src="https://raw.githubusercontent.com/Bimbalacom/Graphical-elements/master/v2/500x500.png"
+                    alt="Logo" class="absolute bottom-0 left-0 mb-4 ml-4 h-16 w-16" />
+            </div>
+        </div>
+    </section>
+
+    <script>
+        // Initialize the history array
+        var nameHistory = JSON.parse(localStorage.getItem("nameHistory")) || [];
```

Comment:
Let's limit it to the latest 5 only. It could become a significantly huge list while the user is still unable to clear it.
Anyway, I think last 5 is enough.

Reply:
It's already [done](https://github.com/Bimbalacom/Desktop/pull/5/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R84-R86). My bad.